composer.json

```diff
@@ -22,13 +22,11 @@
     },
     "require": {
         "php": "^7.4 || ~8.0.0 || ~8.1.0",
-        "laminas/laminas-code": "^4.4",
```

Comment:
Not sure why this was here: it isn't used in the package at all, and therefore should not be in dependencies anyway.